src/common/ErrorHandling.spec.ts

```diff
@@ -0,0 +1,107 @@
+import supertest from 'supertest'
+import express from 'express'
+import { buildURL } from '../../spec/utils'
+import { app } from '../server'
+import { HTTPError } from './HTTPError'
+import { asyncHandler } from './asyncHandler'
+
+const errorMessage = 'anErrorMessage'
+const errorStatusCode = 400
+const errorData = { id: 'anId' }
+
+const handlerWithHttpError = () => {
+  throw new HTTPError(errorMessage, errorData, errorStatusCode)
+}
+const handlerWithCommonError = () => {
```

Comment:
Are both handlerWithHttpError and handlerWithCommonError supposed to throw an HTTPError?

Reply:
They're not! I'll fix those issues

Reply:
Done, fixed.